### DISCUSSION TITLE
[erledigt] F!B 6660 Wlan funktioniert nicht / kein WireGuard in AVM Ui

### DISCUSSION THREAD

**Discussion by Prisrak1** [Freetz-NG/freetz-ng#796] [Allgemein] [answered]
Ubuntu 22.04.2 LTS (GNU/Linux 5.15.0-72-generic x86_64)
 freetz-ng 21630MOA-ef9c8a2d3 master 2023-05-19    

die WLAN LED blinkt ganz schnell. Somit funktioniert WLAN gar nicht. Im WLAN
Funknetz ist alles ausgegraut. Was ist denn da falsch gelaufen?

Beim Bauen des Freetz Images 6660 v.7.50 gibt es kein WireGuard in AVM Ui, obwohl es beim Erstellen nicht herausgenommen wurde. Wie kann man das beheben?

Removal patches --->
[ ] Remove MyFritz │ │
│ │ [ ] Remove DDNSD │ │
│ │ [ ] Remove VPN │ │
│ │ [ ] Remove WireGuard (NEW)
unter Packages unter Freetz, gehts schon.

[*] wireguard-tools 
│ │ [X] wireguard-linux-compat

**Comment by fda77** [comment]
Ich würde vermuten dass das ein problem von avm ist

**Comment by SaMMy-lacht** [comment]
Dann frage ich mich aber wie es kommt das da z. B. WLAN funktioniert 
https://www.digital-eliteboard.com/threads/oscam-tests-bezueglich-oscam-auf-der-6591-6660-6690-gesucht-fw-7-2-und-7-5.522593/post-4154223

**Reply by fda77** [reply]
![image](https://github.com/Freetz-NG/freetz-ng/assets/471850/df349fc5-6909-4625-841b-5c98031943f8)

Du erzählst da Käse, im IPPF wurde berichtet dass das auch ein AVM Problem ist: https://www.ip-phone-forum.de/threads/315112/page-2#post-2514411
Daher nutzt halt keine Alpha-Firmware (Labor/Inhaus) von AVM sondern nur normale "Releases". Die sind zwar auch nicht "stable" aber ändert sich deren Fehlerumfang nicht jeden Tag mit einer neuen Version
Davon abgesehen ist Internet was anderes wie Wlan

**Comment by SaMMy-lacht** [comment]
Es geht hier um das WLAN.
Da schreibt prisrak
Als positiv ist zu verzeichnen, das WLAN LED leuchtet bei beiden Versionen dauerhaft und blinkt nicht mehr, wie bei zuvor gehenden.

**Reply by SaMMy-lacht** [reply]
Und beide Test Images da sind Releases die mit freetz bearbeitet wurden.

**Comment by SaMMy-lacht** [comment]
Und zum Prob des kein WireGuard in AVM Ui. Habe ich prisrak schon mal was gesagt.
Sobald ein Gerät als IP-Client oder als Repeater genutzt wird. Wird es nicht mehr angeboten. Das Gerät muss mit Zugangsdaten und Internet versorgt sein. Dann bekommt man es in dem Gerät angezeigt.

**Reply by SaMMy-lacht** [reply]
[Bildschirmaufzeichnung vom 21.05.2023, 02:39:28.webm](https://github.com/Freetz-NG/freetz-ng/assets/126684514/873a1664-3670-4449-a103-c81b7fa51ea9)

**Reply by fda77** [reply]
Das hat avm wohl so gemacht weil man auf einem "Client" Gerät keine Freigabe machen kann sondern dies auf dem Router machen müsste.  Würde in die Avm-Logik passen. Es gibt auch Geräte bei denen Lan4-Gast nur geht wenn man DSL für Internet nutzt und nicht WAN(bzw LAN1)...  Das kann ich aber nur vermuten da ich diese Features von avm noch nie genutzt hab.

**Comment by Prisrak1** [comment, marked as answer]
Super!  WD bleibt drin.

Anscheinend wurde das Problem zumindest in diesem Fall gelöst. Bis jetzt startet die Box nicht mehr immer wieder neu, wie früher und WLAN funktioniert.

Zu WireGuard: bei mir ist die Fritz!Box 6660 zurzeit über lan 1 mit der Fritz Box 6690 verbunden.
So erscheint die Einstellung VPN WireGuard unter Freigabe erst, nachdem wen man bei der Zugangsart im Internet hin und her schaltet. (Internetzugang über den Kabelanschluss / Internetzugang über "LAN 1")

**Reply by fda77** [reply]
Avm halt...
